src/cmd/cmd.cpp

```diff
@@ -348,16 +355,19 @@ int main(int argc, char **argv)
         options.target_url.append(account->davPath());
     }
 
-    QUrl url = QUrl::fromUserInput(options.target_url);
+    QUrl baseUrl = QUrl::fromUserInput(options.target_url);
+
+    // take only the base URL (without the path)
+    baseUrl = baseUrl.toString().remove(baseUrl.path());
```

Comment:
this will break if the user is deployed to a path other than the root path of the webserver
imagine something like `https://www.example.com/cloud/